plugins/statistics.koplugin/main.lua

```diff
@@ -77,15 +77,18 @@ local ReaderStatistics = Widget:extend{
     data = nil, -- table
 }
 
-function ReaderStatistics:isDocless()
+function ReaderStatistics:isDocless(preserve_finished_books)
     return self.ui == nil or self.ui.document == nil or self.ui.document.is_pic == true
+        or (preserve_finished_books and self.settings.preserve_finished_books
+            and self.ui.doc_settings:readSetting("summary").status == "complete")
```

Comment:
Feels a bit like a quick hack :)
(and ugly in the last chunk of this PR:
```lua
    if self:isDocless() or not self.settings.is_enabled then return end
    if not self:isDocless(true) then...
```
)
May be the isDocLess() stuff should be remade so it's just a flag, set in :init() or onReadSettings(), and not this whole bunch of checks each time we onPageUpdate.

Reply:
OK, so, either:
- you don't care about a change of that status while the book is opened, and keep using its state when it was opened (I'm totally fine with that)
- you send a new Event("BookStatusChanged") and an handler in statistics (is it really worth the work? :)

Reply:
And what is the profit?
`readSetting` costs nothing.
All `isDocless(true)` calls will remain in place.

Reply:
I wanted to avoid reading all 3 thousands lines, but it is funny from the beginning
https://github.com/koreader/koreader/blob/5b5b4d9ebc970ee80d49adf66021d71af15b317c/plugins/statistics.koplugin/main.lua#L104-L108